packages/contracts-bedrock/src/L2/SuperchainERC20Factory.sol

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+import { ISemver } from "src/universal/ISemver.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+import { IL2ToL2CrossDomainMessenger } from "src/L2/IL2ToL2CrossDomainMessenger.sol";
+import { ICrossDomainMessenger } from "@openzeppelin/contracts/vendor/optimism/ICrossDomainMessenger.sol";
+import { BeaconProxy } from "@openzeppelin/contracts/proxy/beacon/BeaconProxy.sol";
+import { CREATE3 } from "@rari-capital/solmate/src/utils/CREATE3.sol";
```

Comment:
this repo is incorrect, use this one: <https://github.com/transmissions11/solmate>

Reply:
the repo remapping is using `'@rari-capital/solmate/=lib/solmate'` , should we add another solmate lib?

Reply:
we can also use CREATE3 from `@solady/=lib/solady/src`

Reply:
hmm leave it then, apparently this links to t11s repo somehow